package.json

```diff
@@ -23,6 +23,7 @@
     "gulp-uglify": "^1.5.1",
     "gulp-uglifycss": "^1.0.5",
     "mongoose": "^4.3.4",
+    "ng-dialog": "^0.5.6",
```

Comment:
Let everyone know when the package.json changes since they will need to do an NPM install. This doesn't apply here because we moved the lib into out directory, but good practice to be in.